genmod/commands/score_compounds.py

```diff
@@ -45,8 +45,10 @@
                     is_flag=True,
                     help='If variants are annotated with the Variant Effect Predictor.'
 )
+@click.option('--threshold', type=int, help="Threshold for model-dependent penalty if no compounds with passing score", default=9)
```

Comment:
Feel very free to help me improve these help texts

Reply:
Correctly spelling threshold is already a strong text! 😸